tests/target/impl.rs

```diff
@@ -28,7 +28,7 @@ impl<T> Foo for T
 where
     // comment2
     // blah
-    T: Clone,
+    T: Clone
```

Comment:
trailing comma is trimmed since where clause contains comments. is it fine? @nrc

Reply:
I think we should not remove the comma - we should keep the comma if it is mulit-line and remove it if it is single line. Whether it is multiline because there are multiple clauses or because of comments shouldn't matter.